core/utils/poly-look/src/visualisations/charts/bar-charts/horizontalBarChart/horizontalBarChart.js

```diff
@@ -0,0 +1,329 @@
+import * as d3 from "d3";
+
+import { Chart } from "../../chart";
+
+import "./horizontalBarChart.css";
+const initializingBarHeight = 2;
+const margin = {
+  top: 10,
+  right: 0,
+  bottom: 20,
+  left: 0,
+};
+const barValueMargin = 4;
+const barPaddingProportion = 0.2;
+const groupHeadlineHeight = 14;
+const headlinePadding = 6;
+const barTextBottomMargin = 6;
+const barLabelTopMargin = 2;
+const defaultBarColor = "blue";
+const defaultBarValueColor = "white";
+const defaultBarWidth = 20;
+/**
+ *
+ * @class
+ * @extends Chart
+ * @param {CSS-selector} selector - A CSS selector, where the svg will be attached to
+ * @param {Object[]} data - The data to be visualized as a bubble cluster
+ * @param {string} data[].title - The title/name the bubble has
+ * @param {number} data[].value - The value of the bubble, which corresponds to it's radius
+ * @param {number} [width  = 400] - The width of the svg
+ * @param {number} [height = 400] - The height of the svg
+ * @param {string|callback} [barColor = defaultBarColor] - The color of the bar (callbacks receive event and data)
+ * @param {string} [barValueColor] - The color the values are shown in (default = no values shown)
+ * @param {string} [barWidth] - barWidth is actually bar height since it is a horizontal bar chart but for all intents and purposes it is the bar's width
+ * @param {Object[]} [groups] - This holds the information of what groups are associated with the data (default: dafta is not grouped)
+ */
```

Comment:
Er. This has been disconnected from what it documents, which starts in line 45, right?